assets/js/frontend/everest-forms.js

```diff
@@ -21,6 +22,160 @@ jQuery( function ( $ ) {
 			// Inline validation.
 			this.$everest_form.on( 'input validate change', '.input-text, select, input:checkbox, input:radio', this.validate_field );
 		},
+
+		/**
+		 * Initialize Range Slider Fields.
+		 *
+		 * @since 1.7.0
+		 */
+		init_range_sliders: function() {
+			// Show Range Slider Fields.
+			$( '.evf-field.evf-field-range-slider' ).show();
+
+			// Slider value change handler.
+			$( '.evf-field-range-slider .evf-field-primary-input' ).ionRangeSlider({
+				onFinish: function( elements ) {
+					var field_id = elements.input.closest( '.evf-field' ).data( 'field-id' );
+					everest_forms.setPrefixPostfixTexts( field_id );
+				}
+			})
+			.on( 'change', function( e ) {
+				var new_value = $( this ).val();
+
+				// Update slider input.
+				$( this ).closest( '.evf-field-range-slider' ).find( '.evf-slider-input' ).val( new_value );
+
+				// Update slider handle/highlight/track color.
+				everest_forms.setSliderColors( $( this ).closest( '.evf-field' ) );
+			});
+
+			// Slider input value change handler.
+			$( '.evf-field-range-slider .evf-slider-input' )
+			.on( 'input', function( e ) {
```

Comment:
Event var is passed but never used.

Reply:
Resolved with 38e1bd4fdd984eab4133668204c60ca1ca725b73